lib/ansible/modules/packaging/os/rpm_key.py

```diff
@@ -170,11 +170,15 @@ def execute_command(self, cmd):
         return stdout, stderr
 
     def is_key_imported(self, keyid):
-        cmd=self.rpm + ' -q  gpg-pubkey --qf "%{description}" | ' + self.gpg + ' --no-tty --batch --with-colons --fixed-list-mode -'
+        cmd = self.rpm + ' -q  gpg-pubkey'
+        rc, _, _ = self.module.run_command(cmd, use_unsafe_shell=True)
+        if rc != 0:  # No key is installed on system
+            return False
+        cmd += ' --qf "%{description}" | ' + self.gpg + ' --no-tty --batch --with-colons --fixed-list-mode -'
```

Comment:
We can reuse `stdout` from the previous command here, but not a blocker for merge.